docs/configuration/install.md

```diff
@@ -361,108 +333,169 @@ mysql -i -BN -e 'SELECT @@sql_mode' | grep -E 'ONLY_FULL_GROUP_BY|STRICT_TRANS_T
+```
+apt-get install zip unzip imagemagick openjdk-7-jdk
+```
 
 On Debian/Ubuntu execute:
 ```
-apt-get install ssmtp
+apt-get install zip unzip imagemagick openjdk-8-jdk
 ```
 
 On RHEL/CentOS execute:
 ```
-yum install ssmtp
+yum install zip unzip libxslt ImageMagick java-1.8.0-openjdk
 ```
 
-The configuration file for SSMTP (e.g. ```/etc/ssmtp/ssmtp.conf ```) MAY look as follows:
+Restart the apache webserver after you installed dependencies!
 
-```
-#
-# Config file for sSMTP sendmail
-#
-# The person who gets all mail for userids < 1000
-# Make this empty to disable rewriting.
-root=yourmail@mail.com
-
-# The place where the mail goes. The actual machine name is required no
-# MX records are consulted. Commonly mailhosts are named mail.domain.com
-mailhub=smtp.yourmail.com
+<a name="optional-dependencies"></a>
+### Optional Dependencies
 
-# Where will the mail seem to come from?
-rewriteDomain=yourservername.org
+Depending on your use case, you MAY want to install further dependencies (exact package names vary by distribution and PHP version you are using):
 
-# The full hostname
-hostname=yourserver.example.com
+* php7.3-curl
+* php7.3-xmlrpc
+* php7.3-soap
+* php7.3-ldap
+* ffmpeg
+* mimetex
 
-# Are users allowed to set their own From: address?
-# YES - Allow the user to specify their own From: address
-# NO - Use the system generated From: address
-FromLineOverride=YES
-```
+Restart the apache webserver after you installed dependencies!
 
-<a name="webdav-configuration-optional"></a>
-## WebDAV Configuration (OPTIONAL)
+<a name="get-code"></a>
+## Get the Code
 
-The recommended webserver configuration is either **Apache with mod_php** or **Nginx with PHP-FPM (> 1.3.8)**. Do NOT use **Apache with PHP-FPM** if you use WebDAV.
+You can [download](http://www.ilias.de/docu/goto.php?target=st_229) the latest
+ILIAS release or clone it from [GitHub](https://github.com/ILIAS-eLearning/ILIAS).
+For production use make sure to checkout the latest stable release, not the trunk,
+which is the development branch of the repository.
 
-### WebDAV with Windows Explorer
-Because of a special behaviour in the Windows Explorer, it sometimes fails to add a WebDAV connection with the error code "0x80070043 The Network Name Cannot Be Found".
+We recommend to clone from GitHub and use git to update the code, since this simplifies
+the update to future releases and versions.
 
-To prevent this behaviour, add the following rewrite rules to a .htaccess file in your webroot or to the corresponding section of the configuration of your webserver:
+Clone the code to the web servers docroot (e.g. `/var/www/html`) with the following
+commands:
 
 ```
-RewriteCond %{HTTP_USER_AGENT} ^(DavClnt)$
-RewriteCond %{REQUEST_METHOD} ^(OPTIONS)$
-RewriteRule .* "-" [R=401,L]
+cd /var/www/html/
+git clone https://github.com/ILIAS-eLearning/ILIAS.git .
```

Comment:
We could suggest to use --single-branch here. For me the difference when testing was 500MB of dead waste.